cmake/tpl/SundialsTPL.cmake.template

```diff
@@ -46,38 +42,36 @@ find_package(<TPL> REQUIRED)
 # Section 4: Test the TPL
 # -----------------------------------------------------------------------------
 
-if(<TPL>_FOUND AND (NOT <TPL>_WORKS))
-  # Do any checks which don't require compilation first.
+# Do any checks which don't require compilation first.
 
-  # Create the <TPL>_TEST directory
-  set(<TPL>_TEST_DIR ${PROJECT_BINARY_DIR}/<TPL>_TEST)
-  file(MAKE_DIRECTORY ${<TPL>_TEST_DIR})
+# Try building a simple test
+if(NOT <TPL>_WORKS)
 
-  # Create a CMakeLists.txt file
-  file(WRITE ${<TPL>_TEST_DIR}/CMakeLists.txt "")
+  message(CHECK_START "Testing <TPL>")
```

Comment:
I did not know about the `CHECK_START` message option... cool.

Reply:
This got added in 3.17 so with the update to 3.18 as the minimum we can make some things a little nicer